builder/vmware/iso/config.go

```diff
@@ -75,6 +75,9 @@ type Config struct {
 	// non-functional. See below for more information. For basic VMX
 	// modifications, try `vmx_data` first.
 	VMXTemplatePath string `mapstructure:"vmx_template_path" required:"false"`
+	// This is the name of the initial snapshot created after provisioning and cleanup.
```

Comment:
To update the generated docs from this you should use `go generate ./...`

Reply:
done :)